src/App.js

```diff
@@ -185,9 +185,9 @@ export const App = () => {
           {!mapboxSupported && <ErrorMessage className='webgl-error-message'
             message={t('webGlDisabled')} />}
 
-          {!!map && <SidebarScrollProvider>
```

Comment:
Removing the `map` prerequisite for the Sidebar was pretty much the only step necessary to fulfill the requirements for https://allenai.atlassian.net/browse/ERA-11903.